gcc/rust/typecheck/rust-tyty.h

```diff
@@ -971,11 +971,17 @@ class USizeType : public BaseType
 public:
   USizeType (HirId ref, std::set<HirId> refs = std::set<HirId> ())
     : BaseType (ref, ref, TypeKind::USIZE)
-  {}
+  {
+    //TODO unused; should 'refs' be passed as the last argument to the 'BaseType' constructor call?  Potential change in behavior (if 'refs' is provided by caller)?
```

Comment:
yeah I think these need to pass the reference chain to the base constructor, I think these were added it was added.

Reply:
Lets merge this and we can fix it in a seperate PR